config/vlan.py

```diff
@@ -3,6 +3,7 @@
 
 from time import sleep
 from .utils import log
+from swsssdk import port_util
```

Comment:
no more swsssdk, it is deprecating.

Reply:
Got it. I am sorry, but I cannot find an alternate for port_util in swsscommon or any other places. Are there plans to implement something similar to port_util to swsscommon or somewhere else? I see the some scripts are still using port_util. I think the alternative will be useful.
If not, I am going to implement all the necessary directly inside vlan.py.

Reply:
As I said before, port_util is still can be useful, so, I decided to copy its implementation to utilities_common of sonic_utilities.
I have found all places in sonic_utilities, where port_util is imported from swsssdk and replaced them to import from utilities_common.